public/app/features/alerting/unified/components/MenuItemPauseRule.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { Menu } from '@grafana/ui';
+import { alertRuleApi } from 'app/features/alerting/unified/api/alertRuleApi';
+import { isGrafanaRulerRule, isGrafanaRulerRulePaused } from 'app/features/alerting/unified/utils/rules';
+import { CombinedRule } from 'app/types/unified-alerting';
+
+const { useUpdateRuleMutation } = alertRuleApi;
+
+interface Props {
+  rule: CombinedRule;
+  /**
+   * Method invoked after the request to change the paused state has completed
+   */
+  onPauseChange?: () => {};
+}
+
+/**
+ * Menu item to display correct text for pausing/resuming an alert,
+ * and triggering API call to do so
+ */
+const MenuItemPauseRule = ({ rule, onPauseChange }: Props) => {
+  const { group } = rule;
+  const [updateRule] = useUpdateRuleMutation();
+  const isPaused = isGrafanaRulerRule(rule.rulerRule) && isGrafanaRulerRulePaused(rule.rulerRule);
+  const icon = isPaused ? 'play' : 'pause';
+  const title = isPaused ? 'Resume alert evaluation' : 'Pause alert evaluation';
```

Comment:
nit: perhaps `alert` here is redundant – we also try to disambiguate between "alert rule" and "alert instance" whenever we can to avoid using "alert" as an ambiguous term :)